mysql-test/r/partition_myisam.result

```diff
@@ -79,10 +79,10 @@ a	b
 54	54
 SHOW STATUS WHERE Variable_name LIKE 'Handler%' AND Value >0;
 Variable_name	Value
-Handler_commit	3
-Handler_external_lock	6
+Handler_commit	1
```

Comment:
Why there is this difference?

Reply:
I'm almost sure it's https://bugs.mysql.com/bug.php?id=93430

Reply:
It's 93430. I verified it with MySQL 8.0.13.